src/specs/models/work-request-bodies/WorkRequestClusterCells.v1.yaml

```diff
@@ -8,8 +8,15 @@ properties:
   cellSetName:
     type: string
     description: The name under which the cell set group should be saved.
-  params:
+  config:
     type: object
+    properties:
+      resolution:
+        type: number
+        minimum: 0
+        description: Resolution used to calculate the clustering. Range 0 - 2. Default 0.5 
```

Comment:
If there is a defined range 0-2, can you also add `maximum: 2` as a limit to `resolution`?